src/components/classifier/FieldGuide.js

```diff
@@ -12,11 +13,11 @@ import {
 import EStyleSheet from 'react-native-extended-stylesheet'
 import FieldGuideItemDetail from './FieldGuideItemDetail'
 import FieldGuideItemRow from './FieldGuideItemRow'
-import Icon from 'react-native-vector-icons/FontAwesome'
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
 import PropTypes from 'prop-types';
-import {addIndex, isEmpty, map} from 'ramda'
+import {isEmpty} from 'ramda'
```

Comment:
Not sure if `ramda` use can be replaced with `lodash` (added in PR 572)

Reply:
I see now that `ramda` is used in a lot of different places in a handful of different ways and replacing `ramda` with `lodash` or vice versa is not trivial. Disregard related comments, but maybe add to a someday to look into (if there's overlap between the packages, replacing one with the other).